src/DevChatter.Bot.Web/Pages/BotAdmin.cshtml.cs

```diff
@@ -0,0 +1,48 @@
+using System.ComponentModel;
+using System.Linq;
+using DevChatter.Bot.Core;
+using Hangfire;
+using Hangfire.Server;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace DevChatter.Bot.Web.Pages
+{
+    public class BotAdminModel : PageModel
+    {
+        public void OnGet()
+        {
+
+        }
+
+        public void OnPost()
+        {
+            var runningJobs = JobStorage.Current.GetMonitoringApi()
+                .ProcessingJobs(0, int.MaxValue).ToList();
+
+            bool alreadyRunning = runningJobs.Any(j => j.Value.Job.Type == typeof(BotWorker));
+            if (!alreadyRunning)
+            {
+                BackgroundJob.Enqueue<BotWorker>(bw => bw.Start());
+            }
+        }
+
+        public string Message { get; set; }
```

Comment:
Property after public methods - is it intentional?

Reply:
Nope, not intentional.